be/src/olap/lru_cache.cpp

```diff
@@ -148,29 +143,22 @@ void HandleTable::_resize() {
 
     LRUHandle** new_list = new (std::nothrow) LRUHandle*[new_length];
     memset(new_list, 0, sizeof(new_list[0]) * new_length);
-    for (uint32_t i = 0; i < new_length; i++) {
```

Comment:
If you removed this code, the first node in linked list is not a dummy node any more. Have you check other logics?

Reply:
Why the first node in the linked list is a dummy node before?

Reply:
> Why the first node in the linked list is a dummy node before?

The dummy node was introduced in https://github.com/apache/incubator-doris/pull/4781 , in order to convert the chaining list from singly list to doubly list. But it's not necessary.